### ISSUE TITLE
Can I customize the loss function?

### ISSUE THREAD

**Issue by AbnerAI** [zhixuhao/unet#137]
Hi,ZhiXuHao:
      good evening！
      I found that the loss function here just passes a loss function name, so I want to know if I can completely customize the loss function?
 model.compile(optimizer=Adam(lr=1e-4), loss='binary_crossentropy', metrics=['accuracy'])

**Comment by edwinb-ai** [comment]
Hey @AbnerAi 

>  so I want to know if I can completely customize the loss function?

Yes, you can. As this is a Keras implementation you can use the Keras API to define [custom metrics](https://keras.io/metrics/#custom-metrics) as well as losses.

For example, [here](https://stackoverflow.com/questions/49296099/zero-jaccard-accuracy-in-u-net-implementation-with-keras) is a question of an implementation of the Jaccard index, as well as the loss, in Keras. You can then pass the name of the loss to the model and compile it.

**Comment by ThinhNgVhust** [comment]
I think we should use (loss='mse', optimizer='adam') . Because Unet is likely kind of Autoencoder huh?
